scripts/build-site.sh

```diff
@@ -19,7 +19,7 @@ echo "==========================================================================
 
 if [ ! -d "snooty" ]; then
   echo "Snooty frontend not installed, downloading"
-  git clone -b netlify-poc --depth 1 https://github.com/mongodb/snooty.git 
+  git clone -b --depth 1 https://github.com/mongodb/snooty.git 
```

Comment:
note - i dont mind giving writers main branch, but this will change the flow of release cycles. also can pull the latest tag if we don't want to give them main branch
```
tag=$(git describe --tags --abbrev=0)
echo checkout out ${tag}
git checkout ${tag}
```